addons/mail/static/src/core/common/model/store.js

```diff
@@ -0,0 +1,371 @@
+/* @odoo-module */
+
+import { markup, reactive, toRaw } from "@odoo/owl";
+import { Record } from "./record";
+import { Markup, STORE_SYM, _0, isAttr, isCommand, isMany, isRecord } from "./misc";
+
+export class Store extends Record {
+    [STORE_SYM] = true;
+    /**
+     * Determines whether the inserts are considered trusted or not.
+     * Useful to auto-markup html fields when this is set
+     */
+    trusted = false;
+    UPDATE = 0;
+    /** @type {RecordField[]} */
+    FC_QUEUE = []; // field-computes
```

Comment:
I would move all the technical variables to a `technical: markRaw({ ... })` object to split them clearly from the rest of the actual fields